src/main/webapp/style.css

```diff
@@ -1,7 +1,59 @@
+body, html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.card {
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  transition: box-shadow .25s;
+  border-radius: 2px;
+  margin-top: 30px;
+  width: 500px;
+}
+
+.card-container {
+  margin-right: 30px;
+  margin-left: 30px;
+  flex-shrink: 0;
+  flex-basis: 500px;
+}
+
+.card .card-description {
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 32px;
+}
+
+.card .card-title {
+  font-size: 20px;
+  font-weight: 300;
+  cursor: pointer;
+  line-height: 32px;
+  display: flex;
+}
+
+.card .card-content {
+  padding: 24px;
```

Comment:
When looking and .card and .card-content overall, I'm confused by the margins and padding. There's a lot of empty space on the right, created by making .card-content 400px but card/card-container 500px. There's both padding on .card-content and margin-top on .card. I guess some of this will be made obsolete by combining .card and .card-content.